README.md

```diff
@@ -33,5 +33,5 @@ npm run test
 ```bash
 # Make sure to have proper NPM credentials before publishing.
 # To get credentials, use `npm login`
-./scripts/publish.sh
```

Comment:
how is this going to work if we don't build dist and publish that :o

Reply:
`npm publish` automatically runs the "prepublishOnly" script, which builds dist
And /dist is included in the files

IMO it's much simpler to publish from the root. Then all the path references in package.json ("export", "types", "main") work the same in development as in the published package. (The old publish.sh script didn't handle that, just broke the references)

Reply:
awesome, thanks for the explanation :)